mws/mws.go

```diff
@@ -73,6 +75,30 @@ const (
 // WorkspaceStatusesNonRunnable is a list of statuses in which the workspace is not runnable
 var WorkspaceStatusesNonRunnable = []string{WorkspaceStatusCanceled, WorkspaceStatusFailed}
 
+type GCP struct {
+	ProjectID string `json:"project_id"`
+}
+
+type CloudResourceBucket struct {
+	GCP *GCP `json:"gcp"`
+}
+
+type GCPManagedNetworkConfig struct {
+	SubnetCIDR               string `json:"subnet_cidr"`
+	GKEClusterPodIPRange     string `json:"gke_cluster_pod_ip_range"`
+	GKEClusterServiceIPRange string `json:"gke_cluster_service_ip_range"`
+}
+
+type GCPCommonNetworkConfig struct {
+	GKEConnectivityType     string `json:"gke_connectivity_type"`
+	GKEClusterMasterIPRange string `json:"gke_cluster_master_ip_range"`
+}
+
+type GCPNetwork struct {
+	GCPManagedNetworkConfig *GCPManagedNetworkConfig `json:"gcp_managed_network_config"`
+	GCPCommonNetworkConfig  *GCPManagedNetworkConfig `json:"gcp_common_network_config"`
```

Comment:
Why is GCPCommonNetworkConfig with the type of *GCPManagedNetworkConfig?

Reply:
This is a good one. Will be fixed in prepare/0.3.7 branch